src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.scss

```diff
@@ -15,3 +15,7 @@ table.ceph-chartbox {
   height: 120px;
   width: 120px;
 }
+
+::ng-deep .nav-tabs {
```

Comment:
This should be a global rule.
We also use this in other pages.

Reply:
Ack